sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json

```diff
@@ -1,3093 +1,3258 @@
+                        "name": "interfaceName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "commandName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "payload",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "description": "The Request Payload",
+                            "type": "object"
+                        }
+                    },
+                    {
+                        "$ref": "#/parameters/api-version"
+                    },
+                    {
+                        "name": "connectTimeoutInSeconds",
+                        "in": "query",
+                        "description": "Connect timeout in seconds.",
+                        "required": false,
+                        "type": "integer",
+                        "format": "int32"
+                    },
+                    {
+                        "name": "responseTimeoutInSeconds",
+                        "in": "query",
+                        "description": "Response timeout in seconds.",
+                        "required": false,
+                        "type": "integer",
+                        "format": "int32"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returns the digital twin command response payload",
+                        "schema": {
+                            "description": "The Response Payload",
+                            "type": "object"
+                        },
+                        "headers": {
+                            "x-ms-command-statuscode": {
+                                "description": "Device Generated Status Code for this Operation",
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "x-ms-request-id": {
+                                "description": "Server Generated Request Id (GUID), to uniquely identify this request in the service",
+                                "type": "string"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/devices/{deviceId}/message": {
+            "post": {
+                "summary": "Send a cloud-to-device message to device.",
+                "description": "Send a cloud-to-device message to device. See https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messages-c2d for more information.",
+                "operationId": "SendDeviceCommand",
```

Comment:
Service team needs decide if this should be message or command. Will impact what we put in path. 
(also: pnp calls "methods" as "command", and they have a different behavior than this API)

Reply:
We also need to suggest a sub-client here. For reference, the other c2d messaging related APIs are housed under `HttpRuntime_` sub-client.

Reply:
why should it be called "command" and not message? 

For subclient - what do you mean? Isn't it under messages client already?